dockstore-webservice/src/main/java/io/openapi/api/impl/ToolsApiServiceImpl.java

```diff
@@ -424,13 +428,15 @@ public Response toolsGet(String id, String alias, String toolClass, String descr
             // user didn't provide a descriptor type or the one they provided matches to CWL or WDL
             if (toolClass == null || COMMAND_LINE_TOOL.equalsIgnoreCase(toolClass)) {
                 if (descriptorType == null  || descriptorLanguage == DescriptorLanguage.WDL || descriptorLanguage == DescriptorLanguage.CWL) {
+                    all.addAll(workflowDAO.filterTrsToolsGet(AppTool.class, descriptorLanguage, registry, organization, name, toolname, description, author, checker));
                     all.addAll(toolDAO.findAllPublished());
                 }
 
             }
             if (toolClass == null || WORKFLOW.equalsIgnoreCase(toolClass)) {
                 // filter published workflows using criteria builder
-                all.addAll(workflowDAO.filterTrsToolsGet(descriptorLanguage, registry, organization, name, toolname, description, author, checker));
+                all.addAll(workflowDAO.filterTrsToolsGet(BioWorkflow.class, descriptorLanguage, registry, organization, name, toolname, description, author, checker));
+                all.addAll(workflowDAO.filterTrsToolsGet(Service.class, descriptorLanguage, registry, organization, name, toolname, description, author, checker));
```

Comment:
Do we want this? I thought we weren't surfacing services in TRS yet.

If we do want it, I think there is or will be a new `toolClass` for services, i.e., searching explicitly for workflows should not return services.

Reply:
I didn't think so either, but it actually used to grab services too. I had to add this to fix an existing test.

Reply:
Hmmm, weird. Would have assumed it would be a different `toolClass` too. 
Since no one is presumably relying on it being a workflow (except maybe us :crossed_fingers: hopefully not), this should be changed